App/src/pages/CreateGroup.tsx

```diff
@@ -0,0 +1,102 @@
+import { Box, Container, FormControl, FormLabel, Heading, HStack, Input, Stack, Select } from "@chakra-ui/react";
+import logo from "../assets/logo.png";
+import { useForm } from "react-hook-form";
+import { useNavigate } from "react-router-dom";
+import { GroupCreateDTO, GroupResponseDTO } from "../services/group";
+import { CreateGroupCall } from "../services/group";
+import Button from "../components/Button";
+
+export default function CreateGroup() {
```

Comment:
I don't understand this form at all. There's a lot of fields, but the only thing that needs to be entered is the group name (just look at the DTO for creation).

Reply:
This is a remnant from the mockup, because in the mockup we need to fill in these, but the current backend only requires the name.